src/components/attendee/NavOverlay/SearchOverlay/SearchOverlay.module.scss

```diff
@@ -0,0 +1,79 @@
+@use "scss/attendee/variables.scss"as *;
+@use "scss/attendee/font.scss";
+@use "scss/attendee/space";
+@use "scss/attendee/theme";
+
+.searchOverlayWrapper {
+  height: 100%;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  max-width: 800px;
+  min-height: 100vh;
+  margin: space.empty(10) auto 0 0;
+  padding: space.empty(4) space.empty(3);
+
+  @media (max-width: 768px) {
```

Comment:
[OOS] We should have different variables for these media query width values.

Reply:
I suggest searching for `@media` in the code, there are instances of `min-width: breakpoint.` used as a constant

Reply:
[2ebc1f3](https://github.com/sparkletown/sparkle/pull/2827/commits/2ebc1f31eb8b1121e844036f240cd927eeb6d714)